recipes/ptarcade/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ptarcade" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ptarcade-{{ version }}.tar.gz
+  sha256: bbde7963829705b6f681cb7c7335b0cfd11137cf37786ed05bc0c452b23a29ab
+
+build:
+  skip: true  # [py<39 or win]
```

Comment:
This looks like noarch. Why are you skipping Windows? If that is due to a missing dependency, we still prefer to keep it noarch and let the solver "avoid" the Windows installation. Also, when that dependency is available on Windows, this package would be ready without any changes.

Reply:
It seems that the solver couldn't avoid the Windows installation, see the log here https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=762403&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf.

There are missing dependencies for Windows.

Reply:
So, the scenario I explained above, it is fine to make it noarch then.

Reply:
All builds are failing because there is an upper bound on the Python version for the package. If I add an upper version constraint, the recipe will not pass the linting check. How would you suggest I proceed?

Reply:
Yep. It needs an upper bound to remove py311.

Reply:
My apologies, I misunderstood this check https://github.com/conda-forge/staged-recipes/pull/23678#issuecomment-1679047319. This wasn't for `noarch`.